services/account-without-juggler/repositories/account/datasources/mysqlconn.ts

```diff
@@ -0,0 +1,229 @@
+const debug = require("debug")("accountsansjuggler");
+const mysql = require("mysql");
+import {
+  CrudConnector,
+  EntityData
+} from "loopback-next/packages/repository/src/crud-connector";
+import {
+  Class,
+  DataSource,
+  Entity,
+  Filter,
+  ObjectType,
+  Options,
+  Where
+} from "loopback-next/packages/repository";
+
+export class MySqlConn implements CrudConnector {
+  private connection: any;
+
+  constructor(config: Object) {
+    this.connection = mysql.createConnection(config);
+    if (this.connection.state !== "authenticated") {
+      this.connect().then(
+        function(res) {
+          debug("Connection success ", res);
+        },
+        function(err) {
+          debug("err connecting", err);
+        }
+      );
+    }
+  }
+  name: "mysql";
+  interfaces?: string[];
+  connect() {
+    let self = this;
+    return new Promise<void>(function(resolve, reject) {
+      self.connection.connect(function(err: any) {
+        if (err !== null) return reject(err);
+        resolve(err);
+      });
+    });
+  }
+  disconnect() {
+    let self = this;
+    return new Promise<void>(function(resolve, reject) {
+      self.connection.end(function(err: any) {
+        if (err !== null) return reject(err);
+        resolve(err);
+      });
+    });
+  }
+  ping() {
+    let self = this;
+    return new Promise<void>(function(resolve, reject) {
+      self.connection.ping(function(err: any) {
+        if (err !== null) return reject(err);
+        resolve(err);
+      });
+    });
+  }
+
+  public updateAll(
+    modelClass: Class<Entity>,
+    data: EntityData,
+    where: Where,
+    options: Options
+  ): Promise<number> {
+    throw new Error("Not implemented yet.");
+  }
+
+  public create(
+    modelClass: Class<Entity>,
+    entity: EntityData,
+    options: Options
+  ): Promise<EntityData> {
+    let self = this;
+    let sqlStmt = "INSERT INTO " + modelClass.name + " VALUES ";
```

Comment:
I think you should escape `modelCass.name` in order to prevent SQL script injection. (Class names probably cannot be changed so much as to cause security problem, but I can imagine there may be valid class names that cannot be used in SQL without escaping.)